python_modules/dagster/dagster/core/definitions/graph_definition.py

```diff
@@ -187,6 +188,7 @@ def __init__(
         output_mappings: Optional[List[OutputMapping]] = None,
         config: Optional[ConfigMapping] = None,
         tags: Optional[Dict[str, Any]] = None,
+        metadata: Optional[Dict[str, RawMetadataValue]] = None,
```

Comment:
should we be able to pass metadata here? if so does it get merged with metadata passed to `to_job`?